File: Clockio/app/src/main/java/com/natasha/clockio/data/model/AccessToken.java
package com.natasha.clockio.data.model;

import com.google.gson.annotations.SerializedName;

import lombok.Data;

@Data
public class AccessToken {
    @SerializedName(value = "access_token")
    private String accessToken;

    @SerializedName(value = "token_type")
    private String tokenType;

    @SerializedName(value = "refresh_token")
    private String refreshToken;

    @SerializedName(value = "expires_in")
    private int expiresIn;

    private String scope;
    private String jti;
}

File: Clockio/app/src/main/java/com/natasha/clockio/base/model/AccessToken.kt
package com.natasha.clockio.base.model

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class AccessToken (
    @SerializedName("access_token")
    val accessToken: String,

    @SerializedName(value = "token_type")
    val tokenType: String,

    @SerializedName(value = "refresh_token")
    val refreshToken: String,

    @SerializedName(value = "expires_in")
    val expiresIn: Int,

    val scope: String,
    val jti: String

) : Serializable